pkg/client/httputils.go

```diff
@@ -210,13 +192,20 @@ type fileInfo struct {
 	TotalChunks int
 }
 
+type UploadOptions struct {
+	InitialDuration     time.Duration
```

Comment:
let's group these 4 fields using a new struct, i.e `BackOffOpts` as they are meant to be used together.

Reply:
Done